### PR TITLE
Controllers

### PR DESCRIPTION
**Pull request by lucascomamala**
For this pull request, I have created all controllers and views for my Blog app. 

Created URLS for:
- users#index
- users#show
- posts#index
- posts#show

Created controllers for Users and Posts

Created view files for Users and Posts with placeholder HTML

Capture

### CODE REVIEW

**Review by Ibizugbe — COMMENTED** [review]
Hi @lucascomamala 👋🏼, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has some linter errors. Kindly do well to fix these errors and resubmit.
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: spec/requests/posts_spec.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
-RSpec.describe "Posts", type: :request do
-  describe "GET /index" do
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /index' do
     pending "add some examples (or delete) #{__FILE__}"
   end
 end
```
Comment (TedLivist) [inline spec/requests/posts_spec.rb:7]: - [ ] I cannot find any relevant spec tests for the post index and post show requests. This file should contain your posts tests, rather it contains dummy pending test generated with Rails. Kindly add specs here for index and show requests.

- [ ] Kindly ensure you cover the required scenarios, like rendering template, checking for placeholder, and maybe HTTP status.

File: spec/requests/users_spec.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /index' do
+    pending "add some examples (or delete) #{__FILE__}"
+  end
+end
```
Comment (TedLivist) [inline spec/requests/users_spec.rb:7]: - [ ] I cannot find any relevant spec tests for the user index and user show requests. This file should contain your posts tests, rather it contains dummy pending tests generated with Rails. Kindly add specs here for index and show requests.

- [ ] Kindly ensure you cover the required scenarios, like rendering template, checking for placeholder, and maybe HTTP status.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @lucascomamala,

Good job so far!
✔️ Addition of `rspec-rails` in Gemfile
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/requests/posts_spec.rb (hunk 1 above)
Comment (lucascomamala) [reply spec/requests/posts_spec.rb:7]: I made a mistake and submitted the wrong part of the project, this PR is for the controllers themselves not the specs. Apologies

**Review by Davidosky007 — APPROVED** [review]
Hi @lucascomamala 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* All controllers created 👍 
* Well written PR 👍 
* Good looking readme 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**